ios/Classes/NativeBridge/Handlers/RegistrationHandler.swift

```diff
@@ -31,24 +25,23 @@ enum WebSignInType: Int {
     }
 }
 
-class RegistrationHandler: NSObject, BrowserHandlerToRegisterHandlerProtocol, PinHandlerToReceiverProtocol, RegistrationHandlerToPinHanlderProtocol {
+class RegistrationHandler: NSObject, BrowserHandlerToRegisterHandlerProtocol {
 
     var createPinChallenge: ONGCreatePinChallenge?
     var browserRegistrationChallenge: ONGBrowserRegistrationChallenge?
     var customRegistrationChallenge: ONGCustomRegistrationChallenge?
     var browserConntroller: BrowserHandlerProtocol?
-    var deregisterUserHandler = DeregisterUserHandler()
+
     var signUpCompletion: ((Result<OWRegistrationResponse, FlutterError>) -> Void)?
     
-    unowned var pinHandler: PinConnectorToPinHandler?
-    
     unowned var customNotificationReceiver: CustomRegistrationNotificationReceiverProtocol?
-    
-    //MARK:-
+
+    // Should not be needed
```

Comment:
Why?

Reply:
It's currently used in mobileAuth for some odd reason but that will be removed in another story.